app/src/main/java/chat/rocket/android/util/extensions/Image.kt

```diff
@@ -0,0 +1,19 @@
+package chat.rocket.android.util.extensions
+
+import android.graphics.Bitmap
+
+fun Bitmap.getCompressFormat(mimeType: String): Bitmap.CompressFormat {
```

Comment:
This can be simplified as
```kotlin
fun Bitmap.getCompressFormat(mimeType: String): Bitmap.CompressFormat {
    return when {
        mimeType.contains("jpeg") -> Bitmap.CompressFormat.JPEG
        mimeType.contains("webp") -> Bitmap.CompressFormat.WEBP
        else -> Bitmap.CompressFormat.PNG
    }
}
```

Reply:
The same for above.